lib/blueprinter/view.rb

```diff
@@ -16,10 +16,6 @@ def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [
       @sort_by_definition = Blueprinter.configuration.sort_fields_by.eql?(:definition)
     end
 
-    def transformers
-      view_transformers.empty? ? Blueprinter.configuration.default_transformers : view_transformers
-    end
-
```

Comment:
This logic has been moved inside of `ViewColleciton` since we need to aggregate all of the included transformers before deciding whether or not to apply the configured default transformers. See https://github.com/procore-oss/blueprinter/pull/372#discussion_r1436806378 for details.